custom-recipes/timeseries-forecast-1-train-evaluate/recipe.json

```diff
@@ -38,8 +38,8 @@
         },
         {
             "name": "evaluation_forecasts_dataset",
-            "label": "Test dataset",
-            "description": "Test dataset with forecasts used to compute performance metrics",
+            "label": "Evaluation dataset",
```

Comment:
If we name this "evaluation dataset", then we should also rename other parts of the plugin where we say "test". I am personally OK with both, as long as we are coherent. In our Visual ML, the wording is "test set for final evaluation".